### ISSUE TITLE
sharp alternative?

### ISSUE THREAD

**Issue by coolzilj** [relax/relax#203]
Is it possible to replace sharp with some lightweight image processing library? 
It's just painful when updating sharp, it took me half an hour to install and compile all the dependencies.

```
==> Upgrading 1 outdated package, with result:
homebrew/science/vips 8.1.1
==> Upgrading homebrew/science/vips
==> Installing dependencies for homebrew/science/vips: pkg-config, libpng, glib, orc, cairo, gobject-introspection, icu4c, harfbuzz, pango, pygobject3, sqlite, python, graphicsmagick,
==> Installing homebrew/science/vips dependency: pkg-config
```

**Comment by bruno12mota** [comment]
Can be an option at least for dev environment. Do you know some?

**Comment by magalhas** [comment]
It it's as good as sharp but lighter it should be an option for production as well.

**Comment by coolzilj** [comment]
I found these days ago, but have never used them. 
- [lwip](https://github.com/EyalAr/lwip) written in C/C++, no external runtime dependencies
- [jimp](https://github.com/oliver-moran/jimp) written in JavaScript, with zero external or native dependencies.

**Comment by coolzilj** [comment]
I found the benchmark results. 
http://sharp.dimens.io/en/stable/performance/ 
It seems like sharp is 20x faster than others, option for dev would be great.

**Comment by bruno12mota** [comment]
Nice @coolzilj! Yes, having an option for dev only seems ideal

**Comment by lovell** [comment]
Hello, for anyone interested in the dependency-less install of `sharp` on OS X, please subscribe to https://github.com/lovell/sharp/issues/312

**Comment by rsilvestre** [comment]
A good idea could be to use directly [cloudinary](https://cloudinary.com)

**Comment by lovell** [comment]
Experimental support for a runtime dependency-free version of `sharp` on OS X is now available to test - please see https://github.com/lovell/sharp/issues/312#issuecomment-226841804

**Comment by andyhu** [comment]
https://www.npmjs.com/package/images
This one seems to be a good lightweight alternative of sharp. It has prebult binaries for many platforms so in most case there's no need to compile anything.

**Comment by lovell** [comment]
v0.16.0 of sharp no longer requires the installation of the libvips dependency on OS X. This means all Windows (x64), Linux (x64, ARM) and now OS X users are dependency-less at runtime.

The remaining work to also remove the compilation step via the use of `node-pre-gyp` is being tracked at https://github.com/lovell/sharp/issues/186.

**Comment by lovell** [comment]
Quick follow up for issues linked to https://github.com/lovell/sharp/issues/186 - prebuilt `sharp.node` binaries are now available in v0.20.0.
